### ISSUE TITLE
Cannot define accessibilityRole=radio on Pressable components inside a radiogroup

### ISSUE THREAD

**Issue by mohib-a-wasay-sweetgreen** [aryella-lacerda/react-native-accessibility-engine#128]
**Describe the bug**
Cannot define `accessibilityRole=radio` on Pressable components inside a `radiogroup`

**Component under test**
```js
    <View accessibilityRole="radiogroup">
        <Pressable accessibilityRole="radio">Alpha</Pressable>
        <Pressable accessibilityRole="radio">Beta</Pressable>
        <Pressable accessibilityRole="radio">Gamma</Pressable>
    </View>
```

**Expected behavior**
Test should pass.

**Expected behavior**
Accessibility Error.
        If a component is touchable/pressable, we should inform the user that it behaves like a button/link
       ↳  Set the 'accessibilityRole' prop with a value of 'button' or 'link' or 'imagebutton'
     · If a component is touchable/pressable, we should inform the user that it behaves like a button/link
       ↳  Set the 'accessibilityRole' prop with a value of 'button' or 'link' or 'imagebutton'

**Context**
 - React Native version: [e.g. 0.60]
 - OS: [e.g. iOS 8.1]
 - Test runner: [e.g. Jest]
 - Accessibility Engine version: [e.g. 0.30]

**Comment by aryella-lacerda** [comment]
Hi, @mohib-a-wasay-sweetgreen! 

#127 fixes this bug and should be released in a new version shortly. :v:

**Comment by aryella-lacerda** [comment]
A fix for this was released in v0.7.0! 😁
